test/demo/basic.html

```diff
@@ -20,16 +20,6 @@
     <div id="contentDiv" class="openseadragon1"></div>
     <script type="text/javascript">
 
-        const url = new URL(window.location.href);
```

Comment:
Thank you for removing this. The `drawer: drawer` also needs to be removed below.

Reply:
Oh yes I see. Removed now.